src/main/java/ch/njol/skript/lang/function/FunctionReference.java

```diff
@@ -264,8 +271,8 @@ public boolean resetReturnValue() {
 	protected T[] execute(Event e) {
 		// If needed, acquire the function reference
 		if (function == null)
-			function = (Function<? extends T>) Functions.getFunction(functionName);
-		
+			function = (Function<? extends T>) Functions.getFunction(functionName, global ? null : script);
+
 		if (function == null) { // It might be impossible to resolve functions in some cases!
 			Skript.error("Couldn't resolve call for '" + functionName +
```

Comment:
Should we include something in this error for local functions?

Reply:
This error is if the function doesn't exist right? We don't have any way to determine "local" or "global"